docs/tests/conditionals.rst

```diff
@@ -0,0 +1,209 @@
+Conditional  Statements
+=======================
+
+The conditional section of the pavilion config consists of two main
+directives, ``only_if`` and ``not_if``. This documentation will cover
+how to use them as well as details associated with their use in test
+configs.
+
+Why conditional statements? As Pavilion becomes more popularized as an
+HPC test harness, the test suites it runs will theoretically grow. It is
+unrealistic to manage unique test suites for specific machines. By using
+conditional statements users can have one large test suite that can run
+across all systems. If a user has a test that is specific to a unique
+computer architecture, they can simply add 'only_if arch is unique-arch'.
+Conditional Statements can greatly simplify your testing procedure.
+
+
+only_if
+~~~~~~~
+
+The ``only_if`` section of the pavilion test config allow users to
+specify a series of conditions. As the name would suggest, the test
+will run if and only if ALL conditions are met.
```

Comment:
Otherwise, the test run will still be created but will not be run or built, and will be assigned a status of 'SKIPPED'.